src/humanloop/client.py

```diff
@@ -65,10 +85,19 @@ def __init__(
         timeout: typing.Optional[float] = None,
         follow_redirects: typing.Optional[bool] = True,
         httpx_client: typing.Optional[httpx.Client] = None,
+        opentelemetry_tracer_provider: Optional[TracerProvider] = None,
```

Comment:
The customer can specify their existing OT setup, but I have not tested this. We should talk with Gusto before deciding on this.